club/theme.json

```diff
@@ -366,11 +366,6 @@
                     "style": "solid",
                     "width": "0"
                 },
-                "spacing": {
```

Comment:
Just want to make sure this is intentional to remove this? I think it looks better but wasn't sure if it's a part of this PR.